tests/test_arraycreation.py

```diff
@@ -136,3 +136,29 @@ def test_tril(m, k):
     expected = numpy.tril(a, k)
     result = dpnp.tril(ia, k)
     numpy.testing.assert_array_equal(expected, result)
+
+
+@pytest.mark.parametrize("k",
+                         [-6, -5, -4, -3, -2, -1, 0, 1, 2, 3, 4, 5, 6],
+                         ids=['-6', '-5', '-4', '-3', '-2', '-1', '0', '1', '2', '3', '4', '5', '6'])
```

Comment:
Do you really need to test all these cases?
Isn't one instance enough for cases k>0, k=0, k<0 for testing?